scripts/fix.sh

```diff
@@ -36,5 +36,7 @@ run_for_modules run ${GO_CMD} fmt || exit 2
 run_for_module tests bom_fix || exit 2
 bash_ws_fix || exit 2
 
+log_callout "Syncing go toolchain directives"
+run ./scripts/sync_go_toolchain_directive.sh || exit 2
```

Comment:
Would prefer to move subroutines from `./scripts/fix.sh` to makefile targets. As you already have a makefile target `sync-toolchain-directive` why not add it as subtarget to `fix`?

Reply:
> why not add it as subtarget to `fix`?

Yes, it's another option. Either way works for me.

We can even breakdown the existing ./script/fix.sh into a couple of Makefile targets
- fix-go-dependency (in which we will run `mod_tidy_fix` and the `sync_go_toolchain_directive.sh`)
- fix-gofmt
- fix-script-whitespace

It can be discussed & resolved in a separate PR.